5.6.1/run.sh

```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+if [ ! $(ls -A "$SONARQUBE_HOME"/conf) ]; then
```

Comment:
@andreyhsiao First - could you please finally explicitly answer on question posed in https://github.com/SonarSource/docker-sonarqube/pull/45#issuecomment-242339735 : why you want to automate copying?

Second - check of emptiness of directory is flawed:
```
$ docker build -t test 5.6.1
$ docker run --rm -it test
./bin/run.sh: line 5: [: sonar.properties: unary operator expected

$ docker build -t test 5.6.1-alpine
$ docker run --rm -it test
sh: wrapper.conf: unknown operand
```
I believe that correct check is:
```
if [ -z "$(ls -A "$SONARQUBE_HOME"/conf)" ]; then
```